vendor.sh

```diff
@@ -5,7 +5,11 @@ set -x
 
 cd `dirname $0`
 
-duckdir=../duckdb-main
+if [ -z "$1" ]; then
+  duckdir=../duckdb
+else
+  duckdir=../$1
```

Comment:
I'd expect the user to give a directory path relative to the cwd, do we need to move this before the `cd` and use `realpath`?